lib/Rx/Observable.php

```diff
@@ -1849,4 +1850,21 @@ public function throttle($throttleDuration, $scheduler = null)
             return new ThrottleOperator($throttleDuration, $scheduler);
         });
     }
+
+    /**
+     * If the source Observable is empty it returns an Observable that emits true, otherwise it emits false.
+     *
+     * @return AnonymousObservable
+     *
+     * @demo isEmpty/isEmpty.php
+     * @demo isEmpty/isEmpty-false.php
+     * @operator
+     * @reactivex isEmpty
```

Comment:
isEmpty is a variant of [contains](http://reactivex.io/documentation/operators/contains.html), so that should be 
`* @reactivex contains`